packages/editor/src/components/post-featured-image/index.js

```diff
@@ -54,21 +69,24 @@ function PostFeaturedImage( { currentPostId, featuredImageId, onUpdateImage, onR
 						allowedTypes={ ALLOWED_MEDIA_TYPES }
 						modalClass={ ! featuredImageId ? 'editor-post-featured-image__media-modal' : 'editor-post-featured-image__media-modal' }
 						render={ ( { open } ) => (
-							<Button
-								className={ ! featuredImageId ? 'editor-post-featured-image__toggle' : 'editor-post-featured-image__preview' }
-								onClick={ open }
-								aria-label={ ! featuredImageId ? null : __( 'Edit or update the image' ) }>
-								{ !! featuredImageId && media &&
-									<ResponsiveWrapper
-										naturalWidth={ mediaWidth }
-										naturalHeight={ mediaHeight }
-									>
-										<img src={ mediaSourceUrl } alt="" />
-									</ResponsiveWrapper>
-								}
-								{ !! featuredImageId && ! media && <Spinner /> }
-								{ ! featuredImageId && ( postLabel.set_featured_image || DEFAULT_SET_FEATURE_IMAGE_LABEL ) }
-							</Button>
+							<>
```

Comment:
I noticed that when uploads fail, there's no feedback at all, should we show a message? (A snackbar, an inline message liike the placeholder?)

Reply:
Oops, missed the notice UI: https://github.com/WordPress/gutenberg/pull/17486/commits/d8a1623c3f0bfcfef7e911cfa06dc91f5504296c